admin/commands/pmm/server/docker/deps.go

```diff
@@ -27,12 +27,12 @@ import (
 	"github.com/percona/pmm/admin/pkg/docker"
 )
 
-//go:generate ../../../../../bin/mockery -name=Functions -case=snake -inpkg -testonly
+//go:generate ../../../../../bin/mockery -name=functions -case=snake -inpkg -testonly
 
-// Functions contain methods required to interact with Docker.
-type Functions interface {
-	Imager
-	Installer
+// functions contain methods required to interact with Docker.
+type functions interface {
```

Comment:
maybe it's better to rename it to dockerFunctions or something else?

Reply:
the same with other usages of `functions` naming

Reply:
Originally I had it with the name `dockerFunctions` but golint complained `docker.dockerFunctions` stutters :)
I'd prefer `dockerFunctions` as well.